### PR TITLE
Secured chat machine new script added

### PR DESCRIPTION
**Pull request by Dr-blue-cyber**
# secured chat machine with encryption and decryption

Please include a general summary of the change and/or which issue it fixed. 

Fixes #(issue number)

## Type of change

run the black command 

## Checklist:

- [ ] I have gone through the contributing guidelines before making this Pull Request.
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings

### CODE REVIEW

**Comment by rijusougata13** [comment]
hi Please give a  meaning heading of the PR related to your work and also fix linting issue

**Comment by Dr-blue-cyber** [comment]
hii @rijusougata13 @advaita-saha   all things are working plz check out my PR

**Comment by Dr-blue-cyber** [comment]
can this PR can not be merge to ?? @OBITORASU

File: scripts/Secured chat machine/README.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+<h1 align="center"> Chat Machine  </h1>
+
+<h2 align="right">
+
+![python](https://forthebadge.com/images/badges/made-with-python.svg)
+
+</h2>
+
+This is WebAPP which provides Secured Chat messaging feaure with Encryption ans decyption security tool.
```
Comment (OBITORASU) [inline scripts/Secured chat machine/README.md:9]: This is not a webapp but a local gui application

File: scripts/Secured chat machine/README.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+</h2>
+
+This is WebAPP which provides Secured Chat messaging feaure with Encryption ans decyption security tool. 
+
+Base64 encryptions are scheme used in encryptions and decryption
+
+
+## __ABOUT__
+**ChatMachine** is a only suggest how can we achieve our End-to-End encyotion schems thats leads to secures messaging in python application that helps user to maintain our privacy from hackers and thirs party applications
```
Comment (OBITORASU) [inline scripts/Secured chat machine/README.md:15]: Fix all the typos here.

**Review by OBITORASU — CHANGES_REQUESTED** [review]
Left you a review

**Pull request closed without merging**